pkg/agent/route/route_linux_test.go

```diff
@@ -439,7 +441,7 @@ COMMIT
 -A ANTREA-OUTPUT -m comment --comment "Antrea: DNAT local to NodePort packets" -m set --match-set ANTREA-NODEPORT-IP dst,dst -j DNAT --to-destination 169.254.0.252
 :ANTREA-POSTROUTING - [0:0]
 -A ANTREA-POSTROUTING -m comment --comment "Antrea: SNAT Pod to external packets" ! -o antrea-gw0 -m mark --mark 0x00000001/0x000000ff -j SNAT --to 1.1.1.1
--A ANTREA-POSTROUTING -m comment --comment "Antrea: masquerade Pod to external packets" -s 172.16.10.0/24 -m set ! --match-set ANTREA-POD-IP dst ! -o antrea-gw0 -j MASQUERADE
+-A ANTREA-POSTROUTING -m comment --comment "Antrea: masquerade Pod to external packets" -s 172.16.10.0/24 -m set ! --match-set ANTREA-POD-IP dst ! -o antrea-gw0 -j MASQUERADE --random-fully
 -A ANTREA-POSTROUTING -m comment --comment "Antrea: masquerade LOCAL traffic" -o antrea-gw0 -m addrtype ! --src-type LOCAL --limit-iface-out -m addrtype --src-type LOCAL -j MASQUERADE --random-fully
 -A ANTREA-POSTROUTING -m comment --comment "Antrea: masquerade OVS virtual source IP" -s 169.254.0.253 -j MASQUERADE
```

Comment:
I want to highlight this masquerade rule (used by proxyAll) for which we have to way to configure randomized source port mappings. I don't think it is an issue. It also wasn't part of the user feature request. IIRC, this traffic goes through 2 rounds of SNAT: once in OVS and once in the host network. So at this point the port has already been remapped by OVS (hash or random port mapping, which is different from the default iptables SNAT behavior?). One could argue that `--random-fully` would actually be a good default here.

Reply:
> So at this point the port has already been remapped by OVS (hash or random port mapping, which is different from the default iptables SNAT behavior?)

Invalid statement, see https://github.com/torvalds/linux/blob/3d5f968a177d468cd13568ef901c5be84d83d32b/net/openvswitch/conntrack.c#L1092